jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -432,6 +432,14 @@ function configure_drools() {
     JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.drools.server.filter.classes=${DROOLS_SERVER_FILTER_CLASSES}"
 }
 
+function assign_server_location(){
+    local location=$( callSecureKieServer "https://kubernetes.default.svc" "/var/run/secrets/kubernetes.io/serviceaccount" )
```

Comment:
it will always be empty.
plus we don't need to create a new function to set the location and another function to query k8s api. there are functions for it already.

what we need is to improve the configure_server_location one to make the route configuration transparent.

see here: routeName="secure-${routeName}"
the big point is to remove this hardcoded piece and query the k8s api to know if there is a https route in place.

if there is 2 routes exposed, http and https, we might want to rename this KIE_SERVER_USE_SECURE_ROUTE_NAME env for a more intuitive name.

Reply:
Why empty ? callSecureKieServer return the http or https route found using the k8s api if is empty no routes are exposed, do you want to remove configure_server_location and use just callSecureKieServer but configuring envs with both https and http routes ?

Reply:
I am not seeing callSecureKieServer returning anything.

> no you want to remove configure_server_location and
do not add new functions, improve the existing one to remove the "secure" hardcoded from secured route.